### ISSUE TITLE
waiting for solutions

### ISSUE THREAD

**Issue by Clarisa2024** [reuter-group/webnma3#11]
Hello
I need you help because yesterday i run three project am I still
waiting the result. Could you help me?how long could them take?I am
registered as: [clarisae.alvarez@gmail.com](mailto:clarisae.alvarez@gmail.com)
Please I would like to see this results
Regards,

**Comment by phillippesamer** [comment]
Dear Clarisa,

We are really sorry about this delay. We are working to understand what is causing the issue, and trying to get WebNMA up and running as soon as we can. Sorry that we are a bit slow, as we are in the middle of a (1-person) team transition.

Meanwhile, I was hoping that you could use our code from github as it is, and run the exact same software on your computer. I wrote below the steps that we expect the user to follow... For the time being, could you please try that instead?

--

1. [Optional] If you are not using mamba instead of conda, I recommend at least
the following for much faster conflict-resolution and environment setup in conda.
Just run
`conda install -n base conda-libmamba-solver`
`conda config --set solver libmamba`


2. Download the repository, e.g. using
`git clone https://github.com/reuter-group/webnma3.git`

3. Navigate to the new directory that is created, and enter
`conda env create --file environment.yml`

4. Still on that terminal window, activate the new environment and setup webnma:
`conda activate webnma3-env`
`python setup.py install`

Done! Use the --help flag to get a usage example for each command, e.g.
`webnma ca --help`

--

Kind regards,
Phillippe

**Comment by Clarisa2024** [comment]
Dejar Philip.
Thanos for tour answer. Is ir possibly to instalo it in Windows?

**Comment by phillippesamer** [comment]
Dear Clarisa,
From an early stage it was not possible to run it on Windows because of a dependency (MUSTANG) which was only available on unix-like operating systems. You are welcome to try it now, but I am unfortunately not able to confirm it.
Best,
Phillippe

**Comment by Clarisa2024** [comment]
Ok. I dos not understand. What hoy think i could try? Te web one ir the
Windows?

**Comment by phillippesamer** [comment]
I mean we really do not support the python version (that you may run on your machine) on Windows because of that dependency. Still it might be possible to make it work on Windows, only that we cannot test/support it, OK?

**Comment by Clarisa2024** [comment]
Yes!i do have linux in half of my disk, however, i am not used to install
programs. I hope I managed. Maybe you can help me?
BEsides, do you know when you are going to solve the web program problem?

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by phillippesamer** [comment]
Hi Clarissa.

1. I think it might be too long to give instructions to your one particular case like this here, but I can try to help (still, you would need to do some search in the web or, even better, getting help from someone...!). The easiest would be to install the conda infrastructure from [here](https://www.anaconda.com/download/) and follow their installation instructions. Check the first part [here](https://conda.io/projects/conda/en/latest/user-guide/install/linux.html) also. When that is done, you follow our own instructions [here](https://github.com/reuter-group/webnma3#for-macos-linux) to get webnma running on your machine.

2. No, I'm afraid we cannot give a schedule to fix the problem because we don't know what happened, and there is only 1 person to work on this (between many, many other tasks).

Best regards,
Phillippe
